test/clj/quil/snippet.clj

```diff
@@ -14,9 +16,42 @@
 (def tests-in-set 50)
 (def current-test (atom -1))
 
+(defn skip-automated-compare? [snippet-name]
+  (contains? #{"current-frame-rate-target-frame-rate"
+               "noise"
+               "noise-detail"
+               "random-2d"
+               "random-3d"
+               "random"
+               "random-gaussian"
+               "resize-sketch"
+               "save"
+               "time-and-date"} snippet-name))
+
+(defn- compare-images [expected actual difference]
```

Comment:
Please add comment to this function explaining what arguments it expects and what it returns. As I understand `difference` is a name of a file and it writes diff image into that file. That's not very clear from just reading function and its implementation.

Reply:
done